casper/src/main/scala/coop/rchain/casper/CasperConf.scala

```diff
@@ -17,7 +17,7 @@ import scala.util.{Failure, Success, Try}
 
 final case class CasperConf(
     publicKeyBase16: Option[String],
-    privateKey: Option[Either[String, Path]],
+    privateKey: Option[String],
```

Comment:
Is this means that private key path can only be specified as CLI parameter but not in config file? Am I missing something?

Reply:
I have to check if the config file is ok. I tried the command line.